addons/api/addon/generated/models/host-set.js

```diff
@@ -57,6 +57,13 @@ export default class GeneratedHostSetModel extends BaseModel {
   })
   plugin;
 
+  @attr('number', {
+    description:
+      "The number of seconds between the time boundary syncs the hosts in this set using this host set's plugin. If not provided a system determined default is used",
+    defaultValue: 0,
```

Comment:
Do we want a client-side default?

Reply:
open to suggestions! I can't remember if we decided to show the default value.

Reply:
`Setting to -1 will disable syncing for that host set; setting to zero will cause the set to use Boundary's default. The default may change between releases.`, from [host set proto].(https://github.com/hashicorp/boundary/blob/main/internal/proto/local/controller/api/resources/hostsets/v1/host_set.proto#L64).

Reply:
Hmm I'm not convinced the client side needs to specify a default though.  I'm not sure we do it in any other number fields.  And is it something that _could_ be machine generated?  If no, then we can omit from here.  If the only way to know that `0` is meaningful is the description, it cannot be machine generated.

Reply:
What is the difference between `0` and other falsy values here?  @talanknight

Reply:
Great question.  It looks like that comment might be out dated actually.  zero is actually not an accepted value.  It can be either -1, or a positive number.  Setting the field to null will use the Boundary default here.